frontend/public/components/utils/owners-resource-link.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { K8sResourceKind, referenceForOwnerRef } from '../../module/k8s';
+import { ResourceLink } from './index';
+
+export const OwnersResourceLink: React.FC<OwnersResourceLinkProps> = ({resource}) => {
+  const owners = (_.get(resource.metadata, 'ownerReferences') || [])
+    .map((o, i) => <ResourceLink key={i} kind={referenceForOwnerRef(o)} name={o.name} namespace={resource.metadata.namespace} title={o.uid} />);
+  return (<React.Fragment> { owners.length ? <dd>{ owners }</dd> : 'No owner' } </React.Fragment>);
```

Comment:
```suggestion
  return owners.length ? <dd>{ owners }</dd> : <span className="text-muted">No owner</span>;
```

Reply:
Actually, just noticed, but this shouldn't be a `dd` element since it's not inside a `dl`

Reply:
Changed this to span. 
The presentation of Owner in Listpage is different than that of DetailsPage. I passed a flag as props to determine the logic that would show. Hope that is fine with our coding practice?

Reply:
> I passed a flag as props to determine the logic that would show. Hope that is fine with our coding practice?

I wouldn't make it a prop. I'd do this instead:

```jsx
<dd><OwnerReferences resource={obj} /></dd>
```

Then always render a `span`.

Reply:
I got this - <dd><OwnerReferences resource={obj} /></dd> but the UI view for detailspage has owner count and title which is different from that of Listpage. I need a flag to determine what to show.

Reply:
@cyril-ui-developer Let's change the `dt` to just be `Owner` to be consistent with the table. We don't prefix a count in other places on details pages. Having a static `dt` value is more consistent with other fields.

Reply:
Okay. Thanks